editor/gui/editor_bottom_panel.cpp

```diff
@@ -56,6 +56,7 @@ void EditorBottomPanel::_on_tab_changed(int p_idx) {
 }
 
 void EditorBottomPanel::_theme_changed() {
+	remove_theme_style_override("tabbar_background");
 	Ref<StyleBox> bottom_tabbar_style = get_theme_stylebox("tabbar_background", "BottomPanel")->duplicate();
```

Comment:
Due to bottom panel having BottomPanel variation, this can be removed
```suggestion
	Ref<StyleBox> bottom_tabbar_style = get_theme_stylebox("tabbar_background")->duplicate();
```
(unless you use the alternate solution I mentioned)